ajax/libs/choices.js/package.json

```diff
@@ -0,0 +1,76 @@
+{
+  "name": "choices.js",
+  "version": "2.0.2",
+  "description": "A vanilla JS customisable text input/select box plugin",
+  "main": "./assets/scripts/dist/choices.min.js",
+  "scripts": {
+    "start": "node server.js",
+    "css:watch": "nodemon -e scss -x \"npm run css:build\"",
+    "css:build": "npm run css:sass -s && npm run css:prefix -s && npm run css:min -s",
+    "css:sass": "node-sass --include-path scss assets/styles/scss/base.scss assets/styles/css/base.css && node-sass --include-path scss assets/styles/scss/choices.scss assets/styles/css/choices.css",
+    "css:prefix": "postcss --use autoprefixer -b 'last 2 versions' assets/styles/css/*.css -d assets/styles/css/",
+    "css:min": "csso assets/styles/css/base.css assets/styles/css/base.min.css && csso assets/styles/css/choices.css assets/styles/css/choices.min.css",
+    "js:build": "webpack --minimize --config webpack.config.prod.js && webpack --config webpack.config.prod.js",
+    "js:test": "./node_modules/karma/bin/karma start --single-run --no-auto-watch tests/karma.config.js",
+    "js:test_watch": "./node_modules/karma/bin/karma start --auto-watch --no-single-run tests/karma.config.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/jshjohnson/Choices.git"
+  },
+  "author": "Josh Johnson",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/jshjohnson/Choices/issues"
+  },
+  "homepage": "https://github.com/jshjohnson/Choices#readme",

```

Comment:
We can remove `homepage` field because the homepage point to Github repo. We'll use `repository` info first.